documentation/code-reviews.md

```diff
@@ -0,0 +1,67 @@
+# How we review code
+
+We consider code reviews an essential practice of software engineering. Following the collective refinement of the task and one of the team taking up its implementation, the code review is the moment the work becomes collaborative again.
+
+You open a pull request, explaining your approach and choices. Peers invest time in understanding the attempt and providing feedback. We document any emerging guidelines or decisions, do the work to resolve all comments, and integrate the increment into the product.
+
+## Write a pull request
+
+If you believe that the code in your branch meets the applicable acceptance criteria and the definition of done, you create a pull request (PR). If you have just pushed, a handy link will appear on Azure’s ‘Pull requests’ page to create one immediately. Otherwise, you configure the new PR yourself.
+
+### Set a title
+
+Write an English title, avoiding jargon. This phrase has a broad audience as we use PR titles to compile release notes.
+
+### Describe your approach
+
+Clearly explain how you implemented the task. Be brief, e.g. by using bullet points. A reviewer will be able to find more context and details in the accompanying product backlog item.
+
+If there are particular circumstances, original ways of tackling something, or things you have deliberately left out of scope, mention those as well. Instructions like steps to reproduce a bug or specific test data may also save the reviewer time.
+
+### Add reviewers
+
+Make sure that the right people are designated as reviewers, or better yet, set Azure to do this automatically for you. We want each PR to be approved by at least one of the maintainers.
```

Comment:
```suggestion
Make sure that the right people are designated as reviewers. We want each PR to be approved by at least one of the maintainers.
```

Doing this automatically is something we should implement, no? Not the person doing the PR